.github/workflows/build.yml

```diff
@@ -40,23 +43,25 @@ jobs:
         run: git config --global url."https://github.com/".insteadOf "git://github.com/"
 
       # Set up js package
+      # TODO: Update these versions? @v3 (all workflows)
```

Comment:
It would be good to update all of these versions! Do you run into any troubles?

Reply:
We have to use an old node version (that we're looking to update), I imagine will update everything at the same time